src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -1057,18 +1047,30 @@ def getWorkloadCreateArgs():
                                         "validate": lambda x: x > 0},
 
                      # parameters that can be overwritten during assignment
-                     "UnmergedLFNBase": {"default": "/store/unmerged"},
-                     "MergedLFNBase": {"default": "/store/data"},
                      "AcquisitionEra": {"validate": acqname, "optional": False},
                      "ProcessingString": {"validate": procstring, "optional": False},
                      "ProcessingVersion": {"default": 1, "type": int, "validate": procversion},
                      "Memory": {"default": 2300.0, "type": float, "validate": lambda x: x > 0},
                      "Multicore": {"default": 1, "type": int, "validate": lambda x: x > 0},
                      "EventStreams": {"type": int, "validate": lambda x: x >= 0, "null": True},
-                     "Comments": {"default": ""},
+                     "MergedLFNBase": {"default": "/store/data"},
+                     "UnmergedLFNBase": {"default": "/store/unmerged"},
                      "DeleteFromSource": {"default": False, "type": strToBool},
-                     "SubRequestType": {"default": ""}
                     }
+
+        # these arguments are internally set by ReqMgr2 and should not be provided by the user
+        reqmgrArgs = {"Requestor": {"attr": "owner", "optional": False},
+                      "RequestorDN": {"attr": "owner_dn", "optional": False, "null": False},
+                      "RequestName": {"optional": False, "null": False, "validate": identifier},
+                      "RequestStatus": {"optional": False, "validate": lambda x: x == REQUEST_START_STATE},
+                      "RequestTransition": {"optional": False, "type": list},
+                      "RequestDate": {"optional": False, "type": list},
+                      "CouchURL": {"default": "https://cmsweb.cern.ch/couchdb", "validate": couchurl},
```

Comment:
This can be provide by user as optional value. Since we could use testbed for the this. Or Are we automatically set this?